weave/trace_server/clickhouse_trace_server_batched.py

```diff
@@ -1534,7 +1534,11 @@ def _insert(
                 # │    return 1 << (21 - int(log(row_size, 2)))
                 # │ValueError: negative shift count
                 # when we try to insert something that's too large.
-                raise RequestTooLarge("Could not insert record")
+                raise InsertTooLarge(
```

Comment:
Thoughts on copy here, the user should see this.

Reply:
I would maybe put a CTA - if you need help, please email / post here.... etc.

Reply:
Also, if you can make it shorter. Ask gpt